edge-modules/api-proxy-module/readme.md

```diff
@@ -205,8 +203,75 @@ Lastly, all the module image URIs in **any lower layer** should use the domain n
 
 ### Upload blob
 
-TBD
+This section describes how to use the [blob storage module](https://docs.microsoft.com/en-us/azure/iot-edge/how-to-deploy-blob) to [upload support bundle](https://github.com/Azure/iotedge/blob/master/doc/built-in-logs-pull.md). 
+It assumes that the blob storage module has already been deployed. Please follow this [link](https://docs.microsoft.com/en-us/azure/iot-edge/how-to-deploy-blob) for detail on how to deploy it. 
+
+>If you want to upload support bundle, currently, the only supported configuration is when the blob storage module is at the top level. To be able to upload a support bundle, a blob container needs to be created and as of now that is only possible when the blob storage module is at the top level.
+
+The configuration of the API Proxy module at the **top layer** for this scenario is the following
+
+- Environment variables:
+
+    | Name  | Value |
+    | ------------- |  ------------- |
+    |BLOB_UPLOAD_ROUTE_ADDRESS|module_blob:11002|
```

Comment:
is module_blob the name of the blob module? If yes, it may be worth making that explicit. Also, by default the name of the Blob module will be "AzureBlobStorageonIoTEdge" when deployed form the marketplace, so i d recommend switching to this default.